lib/blocks.php

```diff
@@ -16,6 +16,41 @@ function gutenberg_reregister_core_block_types() {
 		return;
 	}
 
+	$block_folders = array(
```

Comment:
Instead of an explicit array, can we just loop through the folder?

Reply:
In the ideal world, I would do it, but some blocks are behind the flag and other call `register_block_from_metadata` in PHP file. Well, we could detect the latter, and blacklist the former. Hmmm, I might open a follow-up. I guess I could use `glob` right?

Reply:
There is also this proposed addition to WordPress core from @aduth in https://core.trac.wordpress.org/ticket/49615:
`register_block_type_args` filter

With that, we could find all `block.json` file to register blocks and use this hook to set `render_callback` separately.